studio/components/interfaces/TableGridEditor/SidePanelEditor/TableEditor/SpreadsheetImport/SpreadsheetImport.utils.ts

```diff
@@ -51,7 +51,7 @@ export const parseSpreadsheet = (
   return new Promise((resolve) => {
     Papa.parse(file, {
       header: true,
-      dynamicTyping: true,
+      dynamicTyping: false,
```

Comment:
@joshenlim when this is set to true, looks like Papa Parse parses dates as unix timestamps. When it does that, this value passes the Number() check and maps the field as an int8 instead of a date. 

I did a bunch of testing and it seems like we're ok to set dynamicTyping to false and handle it ourselves. But would love a double check from you!

Reply:
yeap this should be okay - as long as we're parsing them correctly ourselves
we may want to verify if we left out anything that having `dynamicTyping` as true might be giving us [[ref](https://www.papaparse.com/docs#config)]

e.g keeping numbers as string if they exceed certain limits to preserve their precision

Reply:
btw @saltcod could we just double check if the hypothesis [here](https://github.com/mholt/PapaParse/issues/616#issuecomment-462068363) is correct?

if setting `worker` to `false` instead may help? we may also need to check how much performance gets hit by not using the worker option

Reply:
@joshenlim re: #1 — check that we're handling all of the fields correctly with `dynamicTyping`: false.  

re #2 — played with setting `worker` to `false`  and ran into the same issue. I'm able to upload other big csv files (15MB) no trouble, but it chokes on this one. Maybe it's because there's a lot of columns? It seems to be miscalculating the chunk size. Did a bunch of testing with the new 0.1MB setting I have and that seems to work reliably without any noticeable slowdown in importing.